plugins/modules/pubnub_blocks.py

```diff
@@ -243,7 +243,7 @@
 
 try:
     # Import PubNub BLOCKS client.
-    from pubnub_blocks_client import User, Account, Owner, Application, Keyset  # noqa: F401, pylint: disable=unused-import
+    from pubnub_blocks_client import User
```

Comment:
These imports **might** be necessary to ensure a certain version of `pubnub_blocks_client`. (These types are mentioned in Python docstrings and can eventually be converted to type hints.)

Reply:
Well, despite being published in PyPI, the `pubnub_blocks_client` seems to have been discontinued. The last release was in 2017, and in its PyPI page the download link point to https://github.com/pubnub/pubnub-blocks-client/tarball/1.1.0 but not only that link fails, but the very project https://github.com/pubnub/pubnub-blocks-client/ is also a 404.

This module seems to be the evolution of: https://github.com/pubnub/pubnub-blocks-ansible/blob/master/module/pubnub_blocks.py

Apparently the official package now is https://pypi.org/project/pubnub/ but it has no description in PyPI, and its source is https://github.com/pubnub/python

Reply:
pubnub-blocks-client only have two versions released: 1.0.0 and 1.1.0, and the module requires `>=1.0.0`. I think it is safe to remove those, and IMO this module should be in line for deprecation - at a later time, but not too late.

Reply:
The API URL seems to be responsive, but not sure about all the endpoints.